src/main/scala/cromwell/util/EnhancedExponentialBackoff.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.util
+
+import com.google.api.client.util.ExponentialBackOff
+
+class EnhancedExponentialBackoff(enhancedBuilder: EnhancedExponentialBackoff.Builder) extends ExponentialBackOff(enhancedBuilder) {
+  private var firstCall: Boolean = true
```

Comment:
Warning: not fully baked.  Consuming raw or undercooked code may increase your risk of headaches.

```scala
sealed trait Backoff {
  def backoffMillis: Int
  def next: Backoff
}

final case class InitialBackoff(override val backoffMillis: Int) extends Backoff {
  override def next = ExponentialBackoff(...)
}

final case class ExponentialBackoff(googleBackoff: GoogleExponentialBackoff) extends Backoff {
  override def backoffMillis: googleBackoff.nextBackoffMillis
  override def next = this
}
```